<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>crear</class>
 <widget class="QDialog" name="crear">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QLineEdit" name="inputCodigo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
border: 1px solid #3498DB ;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>codigo</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputNombre">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:10px;
border: 1px solid #3498DB ;</string>
   </property>
   <property name="placeholderText">
    <string>nombre</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCategoria">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <item>
    <property name="text">
     <string>categoria</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>computadoras</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>laptops</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>gadgets</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>software</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>harware</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBoxPrecio">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="maximum">
    <number>9999999</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#2ECC71;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>CREAR</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#E74C3C;
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>CANCELAR</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15px;</string>
   </property>
   <property name="text">
    <string>PRECIO</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;</string>
   </property>
   <property name="text">
    <string>OFERTA</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:green;</string>
   </property>
   <property name="text">
    <string>ENVIO GRATIS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:15px;</string>
   </property>
   <property name="text">
    <string>UNIDADES</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxUnidades">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>170</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>9999999</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
